app/job/rest.py

```diff
@@ -170,22 +181,16 @@ def create_job(service_id):
 
         if not is_test_notification:
             check_sms_daily_limit(service, len(recipient_csv))
+            increment_sms_daily_count_send_warnings_if_needed(service, len(recipient_csv))
 
-    if template.template_type == EMAIL_TYPE:
+    elif template.template_type == EMAIL_TYPE:
         check_email_daily_limit(service, len(list(recipient_csv.get_rows())))
+        scheduled_for = datetime.fromisoformat(form.get("scheduled_for")) if form.get("scheduled_for") else None  # noqa: F821
 
-    if data.get("valid") != "True":
-        raise InvalidRequest("File is not valid, can't create job", 400)
-
-    errors = unarchived_template_schema.validate({"archived": template.archived})
-
-    if errors:
-        raise InvalidRequest(errors, status_code=400)
-
-    if template.template_type == SMS_TYPE and not is_test_notification:
-        increment_sms_daily_count_send_warnings_if_needed(service, len(recipient_csv))
-    elif template.template_type == EMAIL_TYPE:
-        increment_email_daily_count_send_warnings_if_needed(service, len(list(recipient_csv.get_rows())))
+        if scheduled_for is None or not scheduled_for.date() > datetime.today().date():
+            increment_email_daily_count_send_warnings_if_needed(
+                authenticated_service, len(list(recipient_csv.get_rows()))  # noqa: F821
```

Comment:
This is still failing local smoke tests.  `authenticated_service` doesnt seem to have the `id` property this method expects it to have.